Declarations/Constants.ts

```diff
@@ -162,6 +162,8 @@ export enum StatsbeatFeature {
     DISK_RETRY = 1,
     AAD_HANDLING = 2,
     WEB_SNIPPET = 4,
+    NATIVE_METRICS = 8,
```

Comment:
Native metrics will no be in distro, maybe add it as a large number

Reply:
Is there any advantage to adding it as a large number over adding it as 8 now and skipping over that value in the distro?

Reply:
bit is reserved for a feature, not for which sdk it is for.. sdkVersion will tell you which sdk it is used. skipping it make it hard to understand.